pyiges/_version.py

```diff
@@ -1,6 +1,6 @@
 """ pyiges version """
 # major, minor, patch, -extra
-version_info = 0, 1, 3
+version_info = 0, 1, 4
```

Comment:
I need to fix this and not bump version info within the master branch.

```suggestion
version_info = 0, 1, 'dev0'
```